src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-windows`
Test-Connection.Ping.can convert IPv6 addresses to IPv4 with -IPv4 parameter
```powershell
Expected collection @('8.8.8.8', '8.8.4.4') to contain $null, but it was not found.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 125
125:             $result.Address.IPAddressToString | Should -BeIn @('8.8.8.8', '8.8.4.4')
```
Test-Connection.Ping.can convert IPv4 addresses to IPv6 with -IPv6 parameter
```powershell
Expected collection @('2001:4860:4860::8888', '2001:4860:4860::8844') to contain $null, but it was not found.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 134
134:             $result.Address.IPAddressToString | Should -BeIn @('2001:4860:4860::8888', '2001:4860:4860::8844')
```